### PR TITLE
Merge testing

### LINKED ISSUE #25: Fortune input process should update the selected fortune bar entry

### LINKED ISSUE #67: Create E2E test for general user flow throughout the site.
- [x] #93
- [x] #25 
~~- [ ] #94~~
~~- [ ] #95~~
- [x] #96
- [x] #97

### LINKED ISSUE #93: The correct fortunes should be displayed on site load

### LINKED ISSUE #96: Origami with unfolded flap should display text from the fortune bar

### LINKED ISSUE #97: Reset/replay buttons should work correctly
- [x] #102
- [x] Reset fortune button should work correctly

### LINKED ISSUE #99: `horizontally-opened.svg` missing top left fold line

### LINKED ISSUE #102: Restart button should work correctly
- [ ] restart button changes SVG path back to closed
- [ ] the correct elements display on the screen (should be same elements when you enter page for the first time) 
- [ ] the sidebar buttons still have the right text saved on them from localStorage

### PR DESCRIPTION
**Pull request by phentos**

### CODE REVIEW

File: assets/scripts/Origami.js
Comment (phentos) [inline assets/scripts/Origami.js:1]: shouldn't be changing source files on the test suite setup branch -- need to be testing as-is. if these are improvements to the code then a good place for them would be alongside work for #64, or maybe a sync from main needs to be done?

Comment (phentos) [reply assets/scripts/Origami.js:1]: team concluded this approach was acceptable, resolve conversation

**Comment by phentos** [comment]
#67 #94 #95 not complete
#99 not complete

**Comment by adarsh249** [comment]
is this good to merge? we're going to have to deal w the sometimes inconsistent test case

**Pull request merged**